mandatory/2-function-creation.js

```diff
@@ -14,27 +18,62 @@ function isAcceptableUser(userAge, isLoggedIn) {}
   The function should return the new price to be paid (e.g. if the totalPrice is 150, a 5% discount
     is applieds and 142.5 should be returned)
 */
+var discount;
+function applyDiscount(totalPrice) {
+  if(totalPrice>200){
+    discount=10;}
+    else{ discount=5;
+
+    }
+    return  totalPrice-( totalPrice * discount / 100);
```

Comment:
Good effort, can we make this code a bit more readable by formatting your if statement?